src/views/FlowNotify.vue

```diff
@@ -4,10 +4,10 @@
 -->
 <template>
 	<div>
-		<input type="text"
+		<input v-model="inscription"
```

Comment:
Okay... so `this.inscription` is synced with the value of this input.
At the same time the `emitInput` function is called whenever the value of the input is changed.

This is confusing because the `emitInput` function also sets `this.inscription`. So depending on the order the `v-model` sync and the event are executed the result might differ.

I'd suggest to try and do one of the two:
* use `v-model` and watch `inscription` for changes. Call the function from there or emit directly.
* use the event and update the value of `inscription` in the handler function.